frontend/src/components/RecordsList.tsx

```diff
@@ -49,7 +49,7 @@ interface IRecordsListProps {
   setShowDownloadModal: Dispatch<SetStateAction<boolean>>;
   handleDownload: (recordCount: number) => void;
   samplesColDefs: ColDef[];
-  sampleContext?: DashboardRecordContext;
+  sampleContexts?: DashboardSamplesQueryVariables["contexts"];
```

Comment:
You'll see these changes throughout this PR's diffs: singular `sampleContext` => plural `sampleContexts`.

This field was used to filter for Request Samples view, WES Samples view, etc. and previously only needed one field to filter by (e.g. Request Samples view filters by some `igoRequestId` value).

With the ACCESS samples view, we're filtering by both `genePanel` and `baitSet`.